### PR TITLE
CustomView Exercise 3

### PR DESCRIPTION
**Pull request by at-lytruong**

### CODE REVIEW

File: ExCustomView/Exercise3/Exercise3/AvatarView.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    func didClickOnAvatar(view: AvatarView)
+}
+
+class AvatarView: UIView {
+    
+    weak var delegate: AvatarViewDelegate?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        imgView()
```
Comment (at-truongnguyen) [inline ExCustomView/Exercise3/Exercise3/AvatarView.swift]: function name can't be a noun. Please change it.

File: ExCustomView/Exercise3/Exercise3/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        
+        self.window = UIWindow(frame: UIScreen.main.bounds)
```
Comment (at-truongnguyen) [inline ExCustomView/Exercise3/Exercise3/AppDelegate.swift]: We shouldn't use `self` here. Please apply for all

File: ExCustomView/Exercise3/Exercise3/AvatarView.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        
+        let imageView = UIImageView()
+        
+        imageView.frame = CGRect(x: 0, y: 0, width: 100, height: 90)
+        imageView.image = UIImage(named: "1")
+        imageView.contentMode = .scaleToFill
+        //        imgView.backgroundColor = UIColor.black
+        addSubview(imageView)
+        
+        let Namelabel = UILabel()
```
Comment (at-truongnguyen) [inline ExCustomView/Exercise3/Exercise3/AvatarView.swift]: variable name can't start with uppercase. please follow camel rule

File: ExCustomView/Exercise3/Exercise3/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+//  ViewController.swift
+//  Exercise3
+//
+//  Created by Hải Lý on 6/14/17.
+//  Copyright © 2017 Hải Lý. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (at-truongnguyen) [inline ExCustomView/Exercise3/Exercise3/ViewController.swift]: remove unnecessary file. pls!

File: ExCustomView/Exercise3/Exercise3/AppDelegate.swift [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
+    }
+
+    func applicationDidBecomeActive(_ application: UIApplication) {
+        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+    }
+
+    func applicationWillTerminate(_ application: UIApplication) {
+        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    }
```
Comment (at-truongnguyen) [inline ExCustomView/Exercise3/Exercise3/AppDelegate.swift]: and remove unused code also

**Comment by at-hieutran** [comment]
@at-lytruong Fix conflict, please.

**Comment by at-hieutran** [comment]
Good boy !!!

**Pull request merged**